install/README.md

```diff
@@ -74,6 +74,7 @@ Manual installation instructions are available for [these distributions](#suppor
 
 ## Installing on a Cloud Service Platform
 * [Google Compute Engine (GCE)](gce-installation-guide.md)
+* [Amazon Web Services (AWS)](aws-installation-guide.md)
```

Comment:
Nit: You could put AWS first to keep this list sorted ;)

Reply:
You've got it! 😄